src/jemalloc_cpp.cpp

```diff
@@ -173,21 +173,38 @@ operator new[](std::size_t size, std::align_val_t alignment, const std::nothrow_
 
 void
 operator delete(void *ptr) noexcept {
+	LOG("core.free.entry", "ptr: %p", ptr);
```

Comment:
The log entry should match the API / the higher level caller. Maybe "core.operator_delete.entry"?

Similarly for all other places in this file. `free` isn't the API used by the application but only the underlying implementation.

Reply:
Sure. Updated the log messages.